tss-esapi/src/structures/buffers.rs

```diff
@@ -185,6 +169,30 @@ pub mod digest {
     }
 }
 
+pub mod ecc_parameter {
+    buffer_type!(
+        EccParameter,
+        crate::tss2_esys::TPM2_MAX_ECC_KEY_BYTES as usize,
+        TPM2B_ECC_PARAMETER
+    );
+}
+
+pub mod encrypted_secret {
+    named_field_buffer_type!(EncryptedSecret, 256, TPM2B_ENCRYPTED_SECRET, secret);
+}
+
+pub mod id_object {
+    named_field_buffer_type!(IDObject, 256, TPM2B_ID_OBJECT, credential);
```

Comment:
Should we change the name of this to ```IdObject```? The camel case naming of types is usually what is used every else.

Reply:
Done!